src/components/Image.ts

```diff
@@ -105,7 +226,8 @@ export class Image extends Component<ImageProps, ImageChild> {
 										element ${QNS.pic}blipFill {
 											element ${QNS.a}blip {
 												attribute ${QNS.r}embed { $relationshipId },
-												attribute cstate { "print" }
+												attribute cstate { "print" },
+												${extensionList ? '$extensionList' : '()'}
```

Comment:
This could be simplified to simply reference the `$extensionList` XPath variable always, if it is `Node | null` instead of `undefined`. `fontoxpath` will in fact consider XPath `()` (an empty node sequence) the same as JavaScript `null`

Reply:
i'll put apply this real quick